src/test/java/igrad/logic/commands/ClearCommandTest.java

```diff
@@ -0,0 +1,32 @@
+package igrad.logic.commands;
+
+import static igrad.logic.commands.CommandTestUtil.assertCommandSuccess;
```

Comment:
i think you removed the static import `import static seedu.address.testutil.TypicalPersons.getTypicalAddressBook;` as well, please use static imports instead! they're much cleaner.